lib/core-server/package.json

```diff
@@ -31,8 +31,7 @@
   },
   "files": [
     "dist/**/*",
-    "dll/**/*",
-    "types/**/*",
+    "public/**/*",
```

Comment:
We had a default `favicon.ico` in the source directory. I moved it to a `public` directory, so that we can stop using `--copy-files` flag from babel on this package.